src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs

```diff
@@ -547,5 +549,11 @@ internal static bool PreventsSuccessfulDelegateConversion(ImmutableArray<Diagnos
 
             return false;
         }
+
+        internal static ErrorCode GetStaticClassParameterCode(bool isInInterface)
+            => isInInterface ? ErrorCode.WRN_ParameterIsStaticClass : ErrorCode.ERR_ParameterIsStaticClass;
+
+        internal static ErrorCode GetStaticClassReturnCode(bool useWarning)
```

Comment:
nit: why do these parameters have different names..?

Reply:
Because I forgot to rename the other one :)